libs/portals/core/src/components/IntroHeader.tsx

```diff
@@ -49,8 +53,11 @@ export const IntroHeader = ({
       </GridColumn>
       {img && (
         <GridColumn span={['8/8', '3/8']}>
-          <Hidden print={hideImgPrint} below="lg">
-            <Box textAlign="center" padding={[6, 0]}>
+          <Hidden print={hideImgPrint} below={imgHiddenBelow}>
+            <Box
+              textAlign={imgPosition === 'right' ? 'right' : 'center'}
```

Comment:
The conditional seems unnecessary, is there a point to the check? Why not use the `imgPosition` argument directly?

Reply:
oh, right! :D thanks!